tests/test_kernel32/test_handle.py

```diff
@@ -120,3 +122,95 @@ def test_process_dies_before_timeout(self):
         self.assertEqual(
             WaitForSingleObject(hProcess, library.INFINITE),
             library.WAIT_OBJECT_0)
+
+
+class TestGetHandleInformation(TestCase):
+    """
+    Tests for :func:`pywincffi.kernel32.GetHandleInformation`
+    """
+    def _expected_inheritance(self):
```

Comment:
This will return a bool. We need a 0 or 1 integer.
Maybe `return 1 if sys.version_info[:2] < (3,4) else 0`?